packages/hash/integration/src/tests/model/ontology/link-type.model.test.ts

```diff
@@ -21,44 +22,45 @@ const graphApi = createGraphClient(logger, {
   port: graphApiPort,
 });
 
-const accountId = "00000000-0000-0000-0000-000000000000";
+let accountId: string;
+const linkTypeSchema: Omit<LinkType, "$id"> = {
+  kind: "linkType",
+  title: "A link",
+  pluralTitle: "Multiple Links",
+  description: "A link between things",
+};
+
+beforeAll(async () => {
+  accountId = await createTestUser(graphApi, "linktypetest", logger);
```

Comment:
These somewhat weird shortnames are because I'm not sure if we can use `kebab-case`, capitals, numbers, etc.

Reply:
Have updated this, kebab case is apparently allowed. I've also updated the default regex to reflect